_includes/footer.html

```diff
@@ -1,26 +1,85 @@
-<footer class='footer'>
-  <div class='footer-copyright footer-section text-container'>
-    <h1 class='foot-txt'>Made by Lotte Bijlsma with great enthusiasm and some chai lattes
+<footer class="footer">
+  <div class="footer-copyright footer-section text-container">
+    <h1 class="foot-txt">Made by Lotte Bijlsma with great enthusiasm and some chai lattes
     </h1>
   </div>
-  <div class='footer-section social-section'>
-    {% if site.site_theme.behance %}
-      <a href="https://be.net/{{ site.site_theme.behance }}" target="_blank">
-        <p class='socials'>Behance</p>
+  <div class="footer-section social-section">
+    <div class="socials">
+      {% if site.site_theme.behance %}
+        <a href="https://be.net/{{ site.site_theme.behance }}" target="_blank">
+          Behance
+        </a>
+      {% endif %}
+      {% if site.site_theme.dribble %}
+        <a href="https://dribbble.com/{{ site.site_theme.dribble }}" target="_blank">
+          Dribbble
+        </a>
+      {% endif %}
+    </div>
+    <div class="socials">
+      {% if site.site_theme.vimeo %}
+        <a href="https://vimeo.com/{{ site.site_theme.vimeo }}" target="_blank">
+          Vimeo
+        </a>
+      {% endif %}
+      <a href="https://github.com/lottebijlsma" target="_blank">
+        Github
       </a>
-    {% endif %}
-    {% if site.site_theme.dribble %}
-      <a href="https://dribbble.com/{{ site.site_theme.dribble }}" target="_blank">
-        <p class='socials'>Dribbble</p>
-      </a>
-    {% endif %}
-    {% if site.site_theme.vimeo %}
-      <a href="https://vimeo.com/{{ site.site_theme.vimeo }}" target="_blank">
-        <p class='socials'>Vimeo</p>
-      </a>
-    {% endif %}
-    <a href="https://github.com/lottebijlsma" target="_blank">
-      <p class='socials'>Github</p>
-    </a>
+    </div>
   </div>
+  <!-- <div class="footer-social footer-section">
```

Comment:
You can delete this on develop if you no longer need it, but do it after merging this branch.